MapboxVision/Services/Recording/RecordCoordinator.swift

```diff
@@ -33,7 +33,14 @@ final class RecordCoordinator {
     }
 
     private var trimRequestCache = [Int: [VideoTrimRequest]]()
-    private(set) var isRecording: Bool = false
+    var isRecording: Bool {
+        var result = false
+        processingQueue.sync {
+            result = _isRecording
+        }
+        return result
+    }
+    private var _isRecording: Bool = false
```

Comment:
Can we try to rename it to avoid `_` prefix? Generally I'm ok with it, but would prefer to avoid.
Probably something like `isRecordingState` or `isRecordingInternal`?

Reply:
sure

Reply:
done